Source: https://github.com/lxde/lxqt-panel/commit/ec62109e0fa678875a9b10fc6f1975267432712d
Upstream: Yes.
Reason: Build failure about "error: '_explicit' does not name a type" with qt5.8.

diff --git a/./plugin-kbindicator/src/x11/kbdlayout.cpp b/kbdlayout.cpp
index c4aa4e2..883df07 100644
--- a/./plugin-kbindicator/src/x11/kbdlayout.cpp
+++ b/kbdlayout.cpp
@@ -33,8 +33,15 @@
 
 #include <xkbcommon/xkbcommon-x11.h>
 #include <xcb/xcb.h>
+
+// Note: We need to override "explicit" as this is a C++ keyword. But it is
+// used as variable name in xkb.h. This is causing a failure in C++ compile
+// time.
+// Similar bug here: https://bugs.freedesktop.org/show_bug.cgi?id=74080
 #define explicit _explicit
 #include <xcb/xkb.h>
+#undef explicit
+
 #include "../kbdinfo.h"
 #include "../controls.h"
 
